src/tests/unittests/validation/CopyCommandsValidationTests.cpp

```diff
@@ -114,89 +114,110 @@ class CopyCommandTest_B2T : public CopyCommandTest {
 
 // Test a successfull B2T copy
 TEST_F(CopyCommandTest_B2T, Success) {
-    nxt::Buffer source = CreateFrozenBuffer(16 * 4, nxt::BufferUsageBit::TransferSrc);
-    nxt::Texture destination = CreateFrozen2DTexture(16, 16, 5, nxt::TextureFormat::R8G8B8A8Unorm,
+    nxt::Buffer source = CreateFrozenBuffer(256 * 4, nxt::BufferUsageBit::TransferSrc);
+    nxt::Texture destination = CreateFrozen2DTexture(256, 16, 5, nxt::TextureFormat::R8G8B8A8Unorm,
                                                      nxt::TextureUsageBit::TransferDst);
 
     // Different copies, including some that touch the OOB condition
     {
         nxt::CommandBuffer commands = AssertWillBeSuccess(device.CreateCommandBufferBuilder())
-            // Copy 4x4 block in corner of first mip.
-            .CopyBufferToTexture(source, 0, destination, 0, 0, 0, 4, 4, 1, 0)
-            // Copy 4x4 block in opposite corner of first mip.
-            .CopyBufferToTexture(source, 0, destination, 12, 12, 0, 4, 4, 1, 0)
-            // Copy 4x4 block in the 4x4 mip.
-            .CopyBufferToTexture(source, 0, destination, 0, 0, 0, 4, 4, 1, 2)
+            // Copy 64x4 block in corner of first mip.
+            .CopyBufferToTexture(source, 0, 0, destination, 0, 0, 0, 64, 4, 1, 0)
+            // Copy 64x4 block in opposite corner of first mip.
+            .CopyBufferToTexture(source, 0, 0, destination, 192, 12, 0, 64, 4, 1, 0)
+            // Copy 64x4 block in the 64x4 mip.
+            .CopyBufferToTexture(source, 0, 0, destination, 0, 0, 0, 64, 4, 1, 2)
             // Copy with a buffer offset
-            .CopyBufferToTexture(source, 15 * 4, destination, 0, 0, 0, 1, 1, 1, 4)
+            .CopyBufferToTexture(source, 192 * 4, 0, destination, 0, 0, 0, 64, 1, 1, 2)
+            .GetResult();
+    }
+
+    // Copies with a 256-byte aligned row pitch but unaligned texture region
+    {
+        nxt::CommandBuffer commands = AssertWillBeSuccess(device.CreateCommandBufferBuilder())
+            // Unaligned block, aligned row pitch
```

Comment:
nit: the comment above mentions "region" but this one mentions "block". Also the rowpitch is always aligned in this test so we might avoid mentioning it.
Also: nit: for misalignment in the texture can we use even more unaligned numbers like 5, 17, 23 and friends? (The buffer offset should still be aligned at a texel size though.)